R/utils_visr.R

```diff
@@ -24,12 +24,12 @@
 #' cowplot::plot_grid(plotlist = list(p1,p2), align = "none", nrow=2)
 #'
 #' ## align_plots() takes into account legend width
-#' cowplot::plot_grid(plotlist = align_plots(pltlist = list(p1, p2)), align = "none", nrow=2)
+#' cowplot::plot_grid(plotlist = .align_plots(pltlist = list(p1, p2)), align = "none", nrow=2)
```

Comment:
align ="v", axis = "l", ncol = 1 to make explicit what we choose as default

Reply:
Reworking this actually exposed unexpected behaviour that would've otherwise gone unnoticed! Nice
![image](https://user-images.githubusercontent.com/12818242/143183076-147909f0-a86b-493e-8781-6ffd1586a147.png)